src/utilities/idf/WorkspaceObject.cpp

```diff
@@ -1231,6 +1237,10 @@ namespace detail {
     }
     WorkspaceObjectVector candidates = m_workspace->getObjectsByReference(iddObject().references());
     for (const WorkspaceObject& candidate : candidates) {
+      if ((candidate.iddObject().type() == openstudio::IddObjectType::OS_Connection) ||
+          (candidate.iddObject().type() == openstudio::IddObjectType::OS_PortList)) {
+        continue;
+      }
```

Comment:
First workaround I had to do that I didn't anticipate. `WorkspaceObject_Impl::uniquelyIdentifiableByName` would crash at `OS_ASSERT(candidateName)`